src/components/App/App.component.jsx

```diff
@@ -1,56 +1,112 @@
-import React, { useLayoutEffect } from 'react';
+import React from 'react';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
-
-import AuthProvider from '../../providers/Auth';
 import HomePage from '../../pages/Home';
 import LoginPage from '../../pages/Login';
 import NotFound from '../../pages/NotFound';
-import SecretPage from '../../pages/Secret';
-import Private from '../Private';
-import Fortune from '../Fortune';
+import VideoDetailsView from '../../pages/VideoDetailsView';
 import Layout from '../Layout';
-import { random } from '../../utils/fns';
+import { createGlobalStyle } from 'styled-components';
+import { useStoreContext } from '../../state/Store.provider';
 
-function App() {
-  useLayoutEffect(() => {
-    const { body } = document;
+const lightTheme = {
+  body: '#fff',
+  a: '#000',
+  cardBodyHover: '#f8f7f7',
+};
+const darkTheme = {
+  body: 'rgba(33, 33, 33, 0.98)',
+  a: '#fff',
+  cardBodyHover: '#353434',
+};
+const GlobalStyle = createGlobalStyle`
```

Comment:
**Optional**: 

Consider moving these styles to a separate file to keep `App.component.jsx` tidier